ci/infra/testrunner/tests/test_masters.py

```diff
@@ -0,0 +1,13 @@
+import time
+
+
+def test_remove_master(deployment, conf, platform, skuba, cluster_ready, joined_master):
+    initial_masters = skuba.num_of_nodes("master")
+
+    # Remove the master
+    skuba.node_remove(role="master", nr=initial_masters - 1)
+    assert skuba.num_of_nodes("master") == initial_masters - 1
+
+    # Wait before checking pods
+    time.sleep(30)
```

Comment:
Couldn't we pass those 30 seconds as a wait_delay to the wait function inside cluster_ready? Cluster ready method could have an optional parameter for that.

I find it a bit confusing having waits spread in many places.

Reply:
Oh yeah left over from previous work on this I'll remove that.